### DISCUSSION TITLE
Will this API be publicly available at the end of the PoC?

### DISCUSSION THREAD

**Discussion by gkgeorgiev** [camptocamp/oapi-poc#21] [Q&A]
Hi guys, 
Firstly, thank you for developing such API that would be really useful to many of us.

My question is if you will expose this API to private individuals for free (after the PoC) ?

I'm asking because such data source would be handy for some home automation ideas I have - a RedNode module. I could probably reverse-engineer the existing website ... but this API looks much nicer :)

Thx

**Comment by davidoesch** [comment]
We will shut down the poc by the end of November....
And then bring the data stream back to life in the final version --- once in the future 2023/2024
However , some of the data can be accessed operationally via simple non api based approach on

https://data.geo.admin.ch/

Look for the sources beginning with ch.meteoschweiz *

**Comment by marcelgsteiger** [comment]
I'm currently using the open and simple APIs of met.no and forecast.solar in a NodeRed app for optimizing energy flow (solar energy, heatpumps, eMobility) in a building. While the temperature forecasts for switzerland are useful the time resolution for our area is bad - update only once every 4 hours. In particular the cloud forecasts are very unreliable. An open api access to swiss metereological data would certainly help a lot. I'm eagerly looking forward to finding a good "official" source.
best regards
Marcel

**Reply by davidoesch** [reply]
Meteoschweiz is Planning to give Opendata access . Drop a mail to info@meteoschweiz.admin.ch
